engine/src/test/java/com/datatorrent/stram/engine/GenericNodeTest.java

```diff
@@ -277,6 +277,8 @@ public int getCount(boolean reset)
     gn.connectInputPort("ip1", reservoir1);
     gn.connectInputPort("ip2", reservoir2);
     gn.connectOutputPort("op", output);
+    gn.firstWindowMillis = 0L;
```

Comment:
387 line has firstWindowMillis initialized as 0 and here firstWindowMillis is initialized as 0L. Can we use same format?